docs/advanced-config.md

```diff
@@ -89,10 +89,39 @@ export OTEL_PROPAGATORS=b3multi
 
 | System property          | Environment variable     | Default value  | Support | Description |
 | ------------------------ | ------------------------ | -------------- | ------- | ----------- |
-| `otel.traces.sampler`    | `OTEL_TRACE_SAMPLER`     | `always_on`    | Stable  | The sampler to use for tracing.	|
-
-Set `otel.traces.sampler` to `internal_root_off` to drop all traces with root spans where `spanKind` is `INTERNAL`, `CLIENT` or `PRODUCER`. This setting only keeps root spans where `spanKind` is `SERVER` and `CONSUMER`.
+| `otel.traces.sampler`    | `OTEL_TRACES_SAMPLER`     | `always_on`    | Stable  | The sampler to use for tracing.	|
+
+Splunk Distribution of OpenTelemetry Java supports all standard samplers as provided by 
+[OpenTelemetry Java SDK Autoconfigure](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#sampler).
+In addition, we add the following two samplers.
+
+### internal_root_off
+Setting `otel.traces.sampler` to `internal_root_off` will drop all traces with root spans where `spanKind` is `INTERNAL`, `CLIENT` or `PRODUCER`.
+This setting only keeps root spans where `spanKind` is `SERVER` and `CONSUMER`.
+
+### rules
+This sampler allows to ignore individual endpoints and drop all traces that would originate from them.
+Let's take a look at the following example configuration:
+```shell
+export OTEL_TRACES_SAMPLER=rules
+export OTEL_TRACES_SAMPLER_ARG=drop=/healthcheck;default=parentbased_always_on
+```
 
+This configuration will result in all requests to `/healthcheck` endpoint to be excluded from monitoring.
+This includes all requests to downstream services that happen during handling this endpoint by the application.
```

Comment:
```suggestion
For example, the following configuration results in all requests to `/healthcheck` to be excluded from monitoring: 

```shell
export OTEL_TRACES_SAMPLER=rules
export OTEL_TRACES_SAMPLER_ARG=drop=/healthcheck;default=parentbased_always_on
```

All requests to downstream services that happen as a consequence of calling an excluded endpoint are also excluded.
```

Reply:
For some reason, the suggestion feature won't let me include the note below the export snippet.